docs/products/postgresql/howto/use-pg_qualstats.rst

```diff
@@ -0,0 +1,331 @@
+    * :doc:`Aiven CLI client </docs/tools/cli>`
+
+Enable pg_qualstats
+-------------------
+
+You can enable the extension in `Aiven Console <https://console.aiven.io/>`_, using SQL, Aiven API, or Aiven CLI.
+
+Enable in Aiven Console
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. important::
+
+   In `Aiven Console <https://console.aiven.io/>`_, you can enable pg_qualstats at the service level only. To enable the extension on a database or for a user, you need to use SQL.
+
+1. Log in to `Aiven Console <https://console.aiven.io/>`_ and navigate to a desired organization, project, and service.
+2. On the **Overview** page of your service, navigate to **Advanced configuration** > **Configure** > **Add configuration options**.
+3. Find the **pg_qualstats.enabled** parameter, set it to **true**, and select **Save configuration**.
+
+Enable with SQL
+~~~~~~~~~~~~~~~
+
+.. note::
+
+   SQL allows for fine-grained enablement of pg_qualstats: on a database, for a user (role), or for a database-role combination.
+
+Enable on a database
+''''''''''''''''''''
+
+1. :doc:`Connect to your Aiven for PostgreSQL service </docs/products/postgresql/howto/list-code-samples>`.
+
+2. Run the following query:
+
+   .. code-block:: bash
+
+      ALTER DATABASE DATABASE_NAME SET pg_qualstats.enabled = 'yes'
+
+Enable for a user
+'''''''''''''''''
+
+1. :doc:`Connect to your Aiven for PostgreSQL service </docs/products/postgresql/howto/list-code-samples>`.
+
+2. Run the following query:
+
+   .. code-block:: bash
+
+      ALTER ROLE ROLE_NAME SET pg_qualstats.enabled = 'yes'
+
+Enable on a DB for a user
+'''''''''''''''''''''''''
+
+1. :doc:`Connect to your Aiven for PostgreSQL service </docs/products/postgresql/howto/list-code-samples>`.
+
+2. Run the following query:
+
+   .. code-block:: bash
+
+      ALTER ROLE ROLE_NAME IN DATABASE DATABASE_NAME SET pg_qualstats.enabled = 'yes'
+
+Enable with Aiven API
+~~~~~~~~~~~~~~~~~~~~~
+
+You can use `Aiven API <https://api.aiven.io/doc/>`_ to enable pg_qualstats on your service. Call the
+`ServiceUpdate <https://api.aiven.io/doc/#tag/Service/operation/ServiceUpdate>`_ endpoint passing ``pg_qualstats.enabled = 'yes'`` in the ``user_config`` object.
+
+Enable with Aiven CLI
+~~~~~~~~~~~~~~~~~~~~~
+
+You can use the :doc:`Aiven CLI client </docs/tools/cli>` to enable pg_qualstats for your service by running the following command:
+
+.. code-block:: bash
+
+   avn service update -c pg_qualstats.enabled=true SERVICE_NAME
+
+Configure pg_qualstats
+----------------------
+
+Pg_qualstats is pre-configured with default settings but you can modify its configuration parameters in `Aiven Console <https://console.aiven.io/>`_, with SQL, using Aiven API, or Aiven CLI.
+
+.. note::
+
+   A service restart might be triggered after modifying particular configuration parameters. You'll get a warning before a potential service restart so that you can decide if you want to proceed with the parameter modification.
```

Comment:
None of the exposed parameters would trigger a restart: we don't allow the customer to change those (eg, pg_qualstats.max)